pkg/conf/config.go

```diff
@@ -56,6 +56,7 @@ func GetStorageParams(env sys.Env) (Parameters, error) {
 	}
 	params["bucket"] = params["builder-bucket"]
 	params["container"] = params["builder-container"]
+	params["regionendpoint"] = params["endpoint"]
```

Comment:
this is the only code change - the rest are lint fixes, and updates needed because of the updated docker/distribution